src/main/java/org/tuberlin/de/geodata/ShapeToTextConverter.java

```diff
@@ -0,0 +1,121 @@
+package org.tuberlin.de.geodata;
+
+import com.vividsolutions.jts.geom.Geometry;
+import org.geotools.data.DataStore;
+import org.geotools.data.DataStoreFinder;
+import org.geotools.data.FeatureSource;
+import org.geotools.feature.FeatureCollection;
+import org.geotools.feature.FeatureIterator;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+
+/**
+ * Created by gerrit on 05.11.15.
+ */
+public class ShapeToTextConverter {
```

Comment:
This class is just for completeness. The textfile already exists in the repository, so upload the existing file to hdfs will work. It's not necessary to use this method.